src/JSONSettingsRESTAPI/Helpers/SettingLookupHelper.cpp

```diff
@@ -0,0 +1,23 @@
+﻿#include "stdafx.h"
+#include "SettingLookupHelper.h"
```

Comment:
Where is being using this class?

Reply:
This class is used by the 2 endpoints that work with a setting given its "id": SettingsGetEndpoint and SettingsSetValueEndpoint. Notice that it is acts as a helper, so the offered method is static and no instance of this class is required to use it.